src/date-range-picker/calendar/grids/index.tsx

```diff
@@ -162,10 +161,7 @@ export const Grids = ({
     }
   };
 
-  const focusVisible = useFocusVisible();
-
-  const isRangeVisible =
-    (selectedStartDate && selectedEndDate) || (gridHasFocus && !!focusVisible['data-awsui-focus-visible']);
+  const isRangeVisible = (selectedStartDate && selectedEndDate) || gridHasFocus;
```

Comment:
This `focusVisible['data-awsui-focus-visible']` condition does not seem to change anything. Tested the component, the outline displayed as expected